db/migrate/20200409043946_create_classified_listing_categories.rb

```diff
@@ -0,0 +1,14 @@
+class CreateClassifiedListingCategories < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
```

Comment:
no need for this as it's an empty table :)
we also should keep transactions on for `CREATE TABLE`

Reply:
The gem complained.

Reply:
> we also should keep transactions on for CREATE TABLE

I understand the general sentiment, but with the way Rails and our pipeline/deployments work I don't think we can have concurrent DDL statements anyway, right?